geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/utils/GeoJsonInference.scala

```diff
@@ -0,0 +1,292 @@
+/***********************************************************************
+ * Copyright (c) 2013-2018 Commonwealth Computer Research, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License, Version 2.0
+ * which accompanies this distribution and is available at
+ * http://www.opensource.org/licenses/apache2.0.php.
+ ***********************************************************************/
+
+package org.locationtech.geomesa.tools.utils
+
+import java.io.InputStream
+import java.{lang => jl}
+import java.time.format.DateTimeParseException
+
+import com.vividsolutions.jts.geom._
+import com.typesafe.config.{Config, ConfigFactory, ConfigValueFactory}
+import com.fasterxml.jackson.core.{JsonFactory, JsonParser, JsonToken}
+import com.fasterxml.jackson.core.JsonToken._
+import org.geotools.feature.AttributeTypeBuilder
+import org.geotools.feature.simple.SimpleFeatureTypeBuilder
+import org.opengis.feature.`type`.AttributeDescriptor
+import org.opengis.feature.simple.SimpleFeatureType
+import org.locationtech.geomesa.utils.io.PathUtils
+import org.locationtech.geomesa.utils.io.fs.FileSystemDelegate
+import org.locationtech.geomesa.utils.geotools.{AttributeSpec, CRS_EPSG_4326}
+import org.locationtech.geomesa.utils.geotools.SimpleFeatureSpecConfig.TypeNamePath
+import org.locationtech.geomesa.utils.text.DateParsing
+
+import scala.collection.JavaConversions._
+import scala.collection.JavaConverters._
+import scala.reflect.ClassTag
+
+object GeoJsonInference {
```

Comment:
it might be useful to put this in geomesa-utils, if the dependencies work out that way

Reply:
Any of the existing submodules make sense to house it? `conf`? Though it looks like we'd have to pull in the jackson json parser at least

Reply:
hmm, if we need jackson then i'd lean towards not moving it - jackson is a pretty common dependency to have classpath conflicts with i think :P
we can re-asses if we want to use it somewhere else